src/names/applications.cpp

```diff
@@ -175,3 +177,10 @@ DescFromName (const valtype& name, NameNamespace ns)
         }
     }
 }
+
+bool
+IsValidJSON (const std::string& text){
```

Comment:
Please consider renaming this to `IsValidJSONOrEmptyString` as per my other comment.  Doesn't matter much, but will avoid any potential confusion in the future.